<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HttpWeather</class>
 <widget class="QWidget" name="HttpWeather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
       <item>
        <widget class="QLabel" name="timeLb">
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>72</pointsize>
          </font>
         </property>
         <property name="text">
          <string>10:36</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="dataLb">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>2024/02/22</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="weekLb">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>星期四</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="PicLb">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
       <item>
        <widget class="QLabel" name="tempLb">
         <property name="font">
          <font>
           <family>Titillium Web</family>
           <pointsize>72</pointsize>
          </font>
         </property>
         <property name="text">
          <string>23℃</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="areaLb">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>深圳 | 晴朗</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
